ssn/rdf/ontology/core/sosa-observation.ttl

```diff
@@ -274,6 +274,8 @@ sosa:isPropertyOf
   rdfs:isDefinedBy sosa-common: .
 
 sosa:forProperty 
```

Comment:
an owl:inverseOf would be very valuable for the communities ObsProp vocabularies and cwould then ompletely cover https://github.com/w3c/sdw-sosa-ssn/issues/47

Reply:
see 1a93

Reply:
Or should it be inverse of `hasProperty` ??

Reply:
See [a852de4](https://github.com/w3c/sdw-sosa-ssn/pull/213/commits/a852de48844797d65482c8e5498a84ab0f492c55)

Reply:
sorry, was away for some weeks. 

`sosa:hasProcedure` was exactly what I was asking for. Thanks !

Reply:
@sgrellet Change your review to 'approve' ?